bin/fuel-core/src/cli/run.rs

```diff
@@ -196,6 +199,11 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
+    /// Number of cores to use for the parallel executor.
+    #[cfg(feature = "parallel-executor")]
+    #[arg(long = "executor-number-of-cores", env, default_value = "1")]
```

Comment:
```suggestion
    #[arg(long = "executor-core-count", env, default_value = "1")]
```

Reply:
Will do in a followup PR to not block this one

Reply:
sounds good